var/spack/repos/builtin/packages/libuv-julia/package.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack.package import *
+
+
+class LibuvJulia(AutotoolsPackage):
+    """Multi-platform library with a focus on asynchronous IO"""
+    homepage = "https://libuv.org"
+    url      = "https://github.com/JuliaLang/libuv/archive/refs/heads/julia-uv2-1.44.1.tar.gz"
+
+    version('1.44.1', sha256='f931e7825702cbb6d07486d92e5436990cf20f91e2b56d6f759822c0f832b13e')
```

Comment:
I'm not sure this is a stable reference, there have been pushes to this branch recently, after it had been prepared for use in the Julia release, for example https://github.com/JuliaLang/libuv/pull/21

Reply:
it's the best we have right now ;p

Reply:
I think a `commit` reference with a [git source](https://spack.readthedocs.io/en/latest/packaging_guide.html#git) might be more accurate?

Reply:
This isn't referencing a branch, it's referencing a released tarball.  If a 1.44.2 release is made then we will add the version and hash, if the 1.44.1 tarball changes (which should not happen) we will find out when the hash breaks.

Reply:
> This isn't referencing a branch,

`refs/heads/julia-uv2-1.44.1` is a branch.

It seems `https://github.com/JuliaLang/libuv/archive/1b2d16477fe1142adea952168d828a066e03ee4c.tar.gz` works too

Reply:
Ah, dangit, my apologies yes it is.  Still, the hash would tell us if and when that branch changes.  Using the commit hash URL would be less likely to break though.